jest.config.js

```diff
@@ -18,10 +18,12 @@ module.exports = {
   clearMocks: true,
 
   // Indicates whether the coverage information should be collected while executing the test
-  // collectCoverage: false,
+  collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  // collectCoverageFrom: undefined,
+  collectCoverageFrom: [
+    "<rootDir>/src/js/conduit/Router.js"
```

Comment:
@brainthinks have we thought about how another developer would add files as they go along? is this required?

Reply:
changed this to be the src/js directory